packages/Autocomplete/src/index.tsx

```diff
@@ -0,0 +1,84 @@
+import Input from "@blaze-react/input";
+import utils from '@blaze-react/utils';
+import React, { Fragment, useState } from "react";
+
+interface IAutocompleteProps {
+  data: {
+    keyValue: string;
+    filterBy: string[];
+    data: object[];
+  };
+  utils: {
+    uniqueId: (element: any) => string
+  };
+  selected: (...args: any[]) => any;
+  placeholder?: string;
+  children?: any;
+}
+const Autocomplete: React.SFC<IAutocompleteProps> = ({
+  data: { data, filterBy: keys, keyValue },
+  placeholder,
+  selected,
+  utils: {
+    uniqueId
+  },
+  children
+}) => {
+  const [inputValue, setInputValue] = useState<string>('');
+  const [showSelect, setShowSelect] = useState<boolean>(false);
+
+  const handleChange = ({ value }: { value: string }): void => {
+    setShowSelect(true);
+    setInputValue(value)
+    filterByValue(value);
+  };
+
+  const filterByValue = (value: string) => {
+    return data.filter((copy: object) => {
```

Comment:
Don't forget to add return type, this is returning a boolean right? Try to add more types in all the code to leverage all that offers TS.